livelog/extractor/writer.go

```diff
@@ -0,0 +1,40 @@
+package extractor
+
+import (
+	"bytes"
+	"encoding/base64"
+	"io"
+	"regexp"
+)
+
+var re = regexp.MustCompile("#((.*?)#)")
+
+type Writer struct {
+	base io.Writer
+	file []byte
+}
+
+func New(w io.Writer) *Writer {
+	return &Writer{w, nil}
+}
+
+func (e *Writer) Write(p []byte) (n int, err error) {
+	if bytes.HasPrefix(p, []byte("#")) == false {
```

Comment:
good catch, yes, we need to pass in as a byte array.  I recommend moving to a global variable

```
var prefix = []byte("#")
```

also since `#` is probably a more common character, maybe we can use something more unique like an ansi escape sequence since it should also play nicely with the shell when testing locally?

```
prefix = \033[1399;
suffix =  \033[0m
```

just a heads up that the above would work in bash, but in Go the `\033` might be represented as `\x1b` (below) which is a hex representation. but I'm not sure, so this may require some further research.

```
prefix = "\x1b[1399;"
suffix = "\x1b[0m"
```

Reply:
Updates added.